packages/crypto/lib/handlers/transactions/handler.js

```diff
@@ -21,7 +21,7 @@ module.exports = class Handler {
     } else {
       const balance = +(wallet.balance.minus(transaction.amount).minus(transaction.fee)).toFixed()
       const enoughBalance = balance >= 0
-      applicable = (transaction.senderPublicKey === wallet.publicKey) && enoughBalance
+      applicable = (transaction.senderPublicKey.toLowerCase() === wallet.publicKey.toLowerCase()) && enoughBalance
```

Comment:
Not really a fan of calling`.toLowerCase()`everywhere... we need to ensure that it is normalized else it is a bug. Is this even still necessary with the SQLite change?

Reply:
SQLite change does fix the source of the uppercase fields. We should be able to get by for now, albeit that code will still be vulnerable to that defect, hence why I put it in there for good measure.

Reply:
This could be an option on the transaction and wallet models to be 100% the keys are always lower-case.

```js
get senderPublicKey () {
    return this._senderPublicKey
}

set senderPublicKey (value) {
    this._senderPublicKey = value.toLowerCase()
}
```

Reply:
For transaction model, it could work, although there are places where we reference a transaction-like object(such as tests), w/o instantiating a Transaction object.

The Wallet model doesn't even have a constructor that accepts parameters like how it's used in some places.

Look at the test for the handler.js file in this changeset, it creates Transaction-like & Wallet-like objects but never actually constructs them. I have no idea if the same happens in non-test code such that it would render getter/setter unused.